test/index.js

```diff
@@ -1182,6 +1182,68 @@ describe('list', function() {
 
   });
 
+  describe('take', function() {
+
+    it('returns a Nothing if n is greater than collection length', function() {
+      eq(S.take(6, ['a', 'b', 'c', 'd', 'e']), S.Nothing());
+      eq(S.take(6, 'abcde'), S.Nothing());
+    });
+
+    it('returns a Nothing if n is less than 0', function() {
+      eq(S.take(-1, ['a', 'b', 'c', 'd', 'e']), S.Nothing());
+      eq(S.take(-1, 'abcde'), S.Nothing());
+    });
+
+    it('returns an empty collection if n is 0', function() {
+      eq(S.take(0, ['a', 'b', 'c', 'd', 'e']), S.Just([]));
+      eq(S.take(0, 'abcde'), S.Just(''));
+    });
+
+    it('returns Just the first two elements from the collection', function() {
+      eq(S.take(2, ['a', 'b', 'c', 'd', 'e']), S.Just(['a', 'b']));
+      eq(S.take(2, 'abcdefg'), S.Just('ab'));
+    });
+
+    it('returns Just the whole collection if n is equal to array length', function() {
+      eq(S.take(-0, ['a', 'b', 'c', 'd', 'e']), S.Just(['a', 'b', 'c', 'd', 'e']));
```

Comment:
Yeah, I wrote those tests before we decided to make negative numbers return Nothing. Fixed.